entrypoint.sh

```diff
@@ -613,6 +617,12 @@ prepare() { local mode no_tap=1
 		no_tap=0 # we want subtests
 	fi
 
+	cat <<EOF > "${BASH_PROFILE}"
+	export KERNEL_BUILD_DIR="${VIRTME_BUILD_DIR}"
+	export PATH="\$PATH:${VIRTME_TOOLS_BIN_DIR}"
```

Comment:
Detail: you can remove the indentation here not to have it in the generated file.

(or use `<<-EOF` and keep the tabs)

(also, while at it, can you use `\${PATH}` with `{}`, and only use one new line below like in the rest of the file? :) )